### ISSUE TITLE
Project maintenance?

### ISSUE THREAD

**Issue by donpui** [LeastAuthority/winden#193]
Is this project still maintained? Are there anyone who can review and merge changes?

I would be glad to implement some issues registered, as sometimes I use it myself.

**Comment by meejah** [comment]
I would also like to know the answer to this question, as I would be more likely to recommend it if I knew it received updates.

I myself no longer do any contract work for Least Authority so cannot provide any insight.

**Comment by btlogy** [comment]
Hi @donpui, good to hear from you.
Sorry we've not seen this issue earlier (I'm not sure if/how maintainers should be notified by GH).
You can ping them from you PR @LeastAuthority/magic-wormhole (I should get the notification then), and we will be happy to (try) to review.

Meanwhile, we'll be bumping some dependencies to maintain this project: #194

**Comment by btlogy** [comment]
BTW: @meejah still has `Write` permissions in this repository. So he could still review/approve and merge PR like he did before.

**Comment by meejah** [comment]
I do not believe I have ever merged nor done any other work in this repository
